plugin/mutate/source/dextool/plugin/mutate/backend/report/html/package.d

```diff
@@ -632,6 +633,60 @@ string toVisible(MetaSpan.StatusColor s) {
     return format("status_%s", s);
 }
 
+/// DB data for coverage-visualization
+struct CoverageData{
+    bool status;
+    int begin;
+    int end;
+
+    static struct T {int value; bool status;}
+    T[] regions;
+
+    bool[int] lineList;
```

Comment:
I think the type should be `bool[uint]` because that is e.g. what SourceLoc use.